static/app/components/forms/controls/selectControl.tsx

```diff
@@ -175,7 +175,7 @@ function SelectControl<OptionType extends GeneralSelectValue = GeneralSelectValu
   // Unfortunately we cannot use emotions `css` helper here, since react-select
   // *requires* object styles, which the css helper cannot produce.
   const indicatorStyles = useCallback(
-    ({padding: _padding, ...provided}: CSSObject) => ({
```

Comment:
padding is overridden 4px 2 lines down no need to destructure

Reply:
this is also expensive since provided creates a new type of CSSObject minus padding